XLS-0085d-token-escrow/README.md

```diff
@@ -209,15 +208,15 @@ The `MPToken` ledger object is updated as follows:
 
 | Field Name      | JSON Type        | Internal Type | Description                                                                                                                                                  |
 |-----------------|------------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `sfEscrowAmount`        | Object | Amount        | *(Optional)* The total of all outstanding escrows for this issuance.                                |
+| `sfLockedAmount`        | Number | UInt64        | *(Optional)* The total of all outstanding escrows for this issuance.                                |
```

Comment:
Hello Dennis, historically, a `string` type has been used in JSON format to represent a `UINT64` value. This helps with maintaining the requisite precision of the values. For example, javascript's `number` cannot represent all the values within the range of `UINT64`.

Values in the MPT object, such as `sfMaximumAmount`, `sfOutstandingAmount` and `sfMPTAmount` use JSON `string` type and `UINT64` as internal rippled type.